modules/nw-ingress-integrating-route-secret-certificate.adoc

```diff
@@ -0,0 +1,71 @@
+//
+// * ingress/routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-integrating-route-secret-certificate_{context}"]
+= Secure route with external certificates in TLS secrets
+
+:FeatureName: Secure route with external certificates in TLS secrets
+include::snippets/technology-preview.adoc[]
+
+The following workflow describes how to integrate third-party certificate management solutions, using the certificate reference field outlined under API Extensions.
+
+A new `.spec.tls.externalCertificate` field has been added to the route's `.spec.tls` configuration. This field can be used to specify the name of a secret containing the certificate data.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the cluster-admin role.
+
+* You must have generated the serving certificate using third-party systems.
+
+* Make sure the `secret` is categorized as `kubernetes.io/tls`, and include both `tls.key` and `tls.crt` keys within the data or `stringData` field of the secret configuration.
+
+.Procedure
+
+* Create a role in the same namespace as the secret containing the certificate generated.
```

Comment:
Can you please add the following phrase also at the end, explaining why it's required?

`granting the router service account read access.`